### PR TITLE
fix(accessibility): contact panel html structure #ACC-7

### PR DESCRIPTION
**Pull request by BartDyb**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-7" title="ACC-7" target="_blank">SubtaskACC-7</a>  9.1.3.2 Fix contacts panel HTML structure
  </td></table>
  <br />
 

https://wearezeta.atlassian.net/browse/ACC-7

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

It contains all the changes necessary to make the contacts sidebar panel HTML structure accessible.

### CODE REVIEW

File: src/script/components/ServiceList.tsx [hunk 1]
```diff
@@ -48,15 +48,30 @@ const ServiceList: React.FC<ServiceListProps> = ({
   noUnderline,
   services,
 }) => {
+  const handleKeyPress = (event: React.KeyboardEvent<HTMLDivElement>, service: ServiceEntity) => {
+    if (event.key === 'Enter' || event.key === ' ') {
```
Comment (kamil-au) [inline src/script/components/ServiceList.tsx]: for those ones you can use `Util/KeyboardUtil` key comparison or even whole functions like `isSpaceKey`, `isEnterKey`

**Review by kamil-au — APPROVED** [review]

File: src/script/components/ServiceList.tsx (hunk 1 above)
Comment (BartDyb) [reply src/script/components/ServiceList.tsx]: does this apply to React's `event.key` property too?

Comment (kamil-au) [reply src/script/components/ServiceList.tsx]: Just checked, you right predefined methods are only for the `KeyboardEvent` type

Comment (BartDyb) [reply src/script/components/ServiceList.tsx]: I've fixed it by using `KEY` constants instead of "Enter" and "  " strings, thanks

**Review by atomrc — APPROVED** [review]

**Pull request merged**